packages/webpack/README.md

```diff
@@ -14,15 +14,20 @@ LavaMoat Webpack Plugin wraps each module in the bundle in a [Compartment](https
+| emitPolicySnapshot       | If enabled, emits the result of merging policy with overrides into the output folder of webpack build for inspection. The file is not used by the bundle.                                                                                                                                                          | `false`                  |
+| readableResourceIds      | Boolean to decide whether to keep resource IDs human readable (possibly regardless of production/development mode). If false, they are replaced with a sequence of numbers. Keeping them readable may be useful for debugging when a policy violation error is thrown. By default follows the webpack config mode. | `(mode==='development')` |
+| lockdown                 | set configuration for [SES lockdown][]. Setting the option replaces defaults from LavaMoat.                                                                                                                                                                                                                        | reasonable defaults      |
+| HtmlWebpackPluginInterop | Boolean to add a script tag to the html output for ./lockdown file if HtmlWebpackPlugin is in use.                                                                                                                                                                                                                 | `false`                  |
+| inlineLockdown           | array of output filenames to inline lockdown into instead of adding it as a file to the output folder.                                                                                                                                                                                                             |
+| runChecks                | boolean property to indicate whether to check resulting code with wrapping for correctness.                                                                                                                                                                                                                        | `false`                  |
+| diagnosticsVerbosity     | number property to represent diagnostics output verbosity. A larger number means more overwhelming diagnostics output. Setting positive verbosity will enable runChecks.                                                                                                                                           | `0`                      |
+| policy                   | the LavaMoat policy object. (by default policy is loaded from files, this is an override)                                                                                                                                                                                                                          | `undefined`              |
```

Comment:
```suggestion
| `policyLocation`           | Folder to store policy files in.                                                                                                                                                                                                                                                                                   | `./lavamoat/webpack`     |
| `generatePolicy`           | Whether to generate the `policy.json` file. Generated policy is used in the build immediately. `policy-override.json` is applied before bundling, if present.                                                                                                                                                        | `false`                  |
| `emitPolicySnapshot`       | If enabled, emits the result of merging policy with overrides into the output folder of Webpack build for inspection. The file is not used by the bundle.                                                                                                                                                          | `false`                  |
| `readableResourceIds`      | Boolean to decide whether to keep resource IDs human readable (possibly regardless of production/development mode). If `false`, they are replaced with a sequence of numbers. Keeping them readable may be useful for debugging when a policy violation error is thrown. By default, follows the Webpack config mode. | `(mode==='development')` |
| `lockdown`                 | Configuration for [SES lockdown][]. Setting the option replaces defaults from LavaMoat.                                                                                                                                                                                                                        | reasonable defaults      |
| `HtmlWebpackPluginInterop` | Boolean to add a script tag to the HTML output for `./lockdown` file if `HtmlWebpackPlugin` is in use.                                                                                                                                                                                                                 | `false`                  |
| `inlineLockdown`           | Array of output filenames in which to inline lockdown (instead of adding it as a file to the output folder).                                                                                                                                                                                                             |
| `runChecks`                | Boolean property to indicate whether to check resulting code with wrapping for correctness.                                                                                                                                                                                                                        | `false`                  |
| `diagnosticsVerbosity`     | Number property to represent diagnostics output verbosity. A larger number means more overwhelming diagnostics output. Setting a positive verbosity will enable `runChecks`.                                                                                                                                           | `0`                      |
| `policy`                   | The LavaMoat policy object (if not loading from file; see `policyLocation`)                                                                                                                                                                                                                          | `undefined`              |
```

Reply:
thank you